src/tmp.jl

```diff
@@ -240,26 +240,52 @@ function tmp_create_union_tables(connection)
     # These are equivalent to the partitions in time-resolution.jl
     # But computed in a more general context to be used by variables as well
 
+    # Incoming flows
+    DuckDB.execute(
+        connection,
+        "CREATE OR REPLACE TEMP TABLE t_union_in_flows AS
+        SELECT DISTINCT to_asset as asset, year, rep_period, time_block_start, time_block_end
```

Comment:
Also here, okay to use only "SELECT"? Do we have duplicates?

Reply:
For this table it is ok to write only `SELECT` because they are filtered later in the resolution computation, but might as well do it now.
There are probably duplicates because this ignores `from_asset`.